Tools/Cli-LoRa-Device-Provisioning/Tests/LoRaWan.Tools.CLI.Tests.Unit/DeviceProvisioningTest.cs

```diff
@@ -104,7 +108,7 @@ public async Task AddABPDevice(bool deviceExistsInRegistry)
             if (deviceExistsInRegistry)
             {
                 this.registryManager.Verify(c => c.UpdateTwinAsync(
-                        DevEUI.ToString(),
+                        devEUI.ToString(),
```

Comment:
nit: we don't need `.ToString()`, as it is already one.
There are other occurences with `devEUI` (check `appKey` and `appEui` for completeness).